### ISSUE TITLE
Do not open new PR if a PR for a previous version is not merged

### ISSUE THREAD

**Issue by simov** [greenkeeperio/greenkeeper#78]
Is there a way for you to check if a PR is already created and not merged for a particular module?

We have some Greenkeeper PRs that we won't merge any time soon, and the reasons are properly documented inside these PRs. But we still get failing new PRs for newer versions, which is kind of annoying.

**Comment by dbkaplun** [comment]
See also #62

**Comment by boennemann** [comment]
Thanks for reporting this @simov. We currently thinking through several ways of approaching this, and not opening new PRs would be one way.

We'll let you know once something lands and how that works.

**Comment by fundon** [comment]
+1

**Comment by boennemann** [comment]
We started to roll out our auto-closing feature. Details: https://github.com/greenkeeperio/greenkeeper/issues/62#issuecomment-176248343
